VotingApplication/VotingApplication.Web/Api/Controllers/ManageController.cs

```diff
@@ -161,6 +199,20 @@ public void Put(Guid manageId, ManagePollUpdateRequest updateRequest)
             }
         }
 
+
+        private void SendInvitation(Guid UUID, Token token)
+        {
+            String hostUri = WebConfigurationManager.AppSettings["HostURI"];
+            if (hostUri == String.Empty)
+            {
+                return;
+            }
+
+            string message = String.Join("\n\n", new List<string>() { "You've been invited to a poll on Pollster",
+            "Have your say at " + hostUri + "/Poll/#/Vote/" + UUID + "/" + token.TokenGuid });
+
+            _mailSender.SendMail(token.Email, "Have your say", message);
```

Comment:
What if 'token' doesn't have an email? (It's coming from the client, so could be a bad request).